### DISCUSSION TITLE
HTML content suddenly breaking on self-hosted Postal server

### DISCUSSION THREAD

**Discussion by craigrileyuk** [postalserver/postal#2749] [Help with using Postal]
We've had a few sites using a self-hosted Postal server for a while and suddenly a couple of them are having issues with the HTML output in mails.

```
----==_mimepart_65a12a01839f_969b412538d7
Content-Type: multipart/alternative;
 boundary="--==_mimepart_65a12a018276_969b4125374a"
Content-Transfer-Encoding: 7bit


----==_mimepart_65a12a018276_969b4125374a
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: quoted-printable

<div class="3D&quot;body_style&quot;" data-builder="3D&quot;99aa00&quot;" style="3D&quot;font-famil=" y:="" tahoma,="" roboto,="" arial,="" helvetica,="" sans-serif;color:="" #3d4852;font-size:="16px;background-color:" #fff"="">
```

and the rest of the HTML/CSS is broken.

This is the same both on the Postal admin panel under Messages -> HTML and on the email itself.

I've tried updating the postal-php package to the latest version (v2.0.1) but it's the same.

**Comment by willpower232** [comment]
Sounds like you're using the API so it might be worth checking the main content of the message before you give it to Postal in case that has a quote in the wrong place or whatever.

I don't think Postal would have spontaneously changed behaviour so it must be to do with however you're generating the content unless there has been some kind of update to guzzle that did it.
